subsys/mgmt/mcumgr/grp/fs_mgmt/src/hash_checksum_crc32.c

```diff
@@ -8,11 +8,10 @@
 #include <string.h>
 #include <zephyr/sys/crc.h>
 #include <zephyr/fs/fs.h>
-#include <mgmt/mgmt.h>
-#include <fs_mgmt/fs_mgmt_config.h>
-#include <fs_mgmt/fs_mgmt_impl.h>
-#include "fs_mgmt/hash_checksum_mgmt.h"
-#include "fs_mgmt/hash_checksum_crc32.h"
```

Comment:
add this include back

Reply:
Not sure why it's showing 4 lines, just the last line for the hash_checksum_crc32.h include

Reply:
Unresolved